### PR TITLE
Fix resize - Delivers #133812817

### PR DESCRIPTION
**Pull request by jeremyjpark**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -24,18 +24,57 @@ $error-color: #EED6D6;
...
+    }
+  }
 }
 
 .nav-wrapper .navbar-non-link {
   padding: 0 15px;
 }
 
+.side-nav {
+  width: 500px;
```
Comment (susannasouv) [inline app/assets/stylesheets/application.scss:56]: em possible? or nah?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -10,14 +10,20 @@
...
-</div>
+  <%= render "shared/navbar" %>
+
+  <div class="container">
+    <% flash.each do |key, value| %>
+        <div class="alert alert-<%= key %>"><%= value %></div>
+    <% end %>
+  </div>
+
+  <div class="container">
```
Comment (susannasouv) [inline app/views/layouts/application.html.erb:22]: Unsuccessful merge?

File: features/user_auth.feature [hunk 3]
```diff
@@ -70,22 +70,22 @@ Feature: User Authentication with Devise
...
-  Scenario: Creating an account with a non-Berkeley email
-    Given I am on the home page
-    And I follow Sign Up
-    When I fill in "user_email" with "datongtest@sdfsdf.com"
-    And I fill in "user_password" with "hunter2"
-    And I fill in "user_password_confirmation" with "hunter2"
-    And I press "Sign Up"
-    Then I should see Welcome! You have signed up successfully.
-    And I should be a non-Berkeley student user
+#  Scenario: Creating an account with a Berkeley email
```
Comment (susannasouv) [inline features/user_auth.feature]: These tests should not be commented out.

**Review by susannasouv — CHANGES_REQUESTED** [review]
User permissions feature cucumber scenarios should not be commented out.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (jeremyjpark) [reply app/assets/stylesheets/application.scss:56]: em unnecessary as pixel-width for most all phones is wide enough to accomodate.

**Comment by jeremyjpark** [comment]
Resolved the commented out tests.

**Review by susannasouv — APPROVED** [review]
lgtm

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (susannasouv) [reply app/views/layouts/application.html.erb:22]: bitmoji

**Pull request merged**